aspnetcore/blazor/components.md

```diff
@@ -238,22 +238,22 @@ If the value of the `ParentYear` property is changed by selecting the button in
 
 The `Year` parameter is bindable because it has a companion `YearChanged` event that matches the type of the `Year` parameter.
 
-By convention, `<ChildComponent bind-Year="@ParentYear" />` is essentially equivalent to writing,
+By convention, `<ChildComponent @bind-Year="@ParentYear" />` is essentially equivalent to writing,
 
 ```cshtml
-<ChildComponent bind-Year-YearChanged="@ParentYear" />
+<ChildComponent @bind-Year="@ParentYear" @bind-Year:event="YearChanged" />
 ```
 
-In general, a property can be bound to a corresponding event handler using `bind-property-event` attribute.
+In general, a property can be bound to a corresponding event handler using `@bind-property:event` attribute.
```

Comment:
This isn't descriptive enough. This used to be one attribute, and now it's two so it requires some explanation.